src/main/resources/liquibase/harness/change/expectedSql/cockroachdb/20.2/modifyDataType.sql

```diff
@@ -0,0 +1,9 @@
+INVALID TEST
```

Comment:
@rafiss i ignored this test for 20.2, if that's not good you can create different modifyDataType.xml in changelogs/cockroachdb/20.2 folder to verify increasing precision that is supported for this version.

Reply:
this looks good, thanks!